test-helpers/src/docker_compose.rs

```diff
@@ -98,22 +103,66 @@ impl DockerCompose {
     /// * `count` - The number of times the regular expression should be found.
     ///
     /// # Panics
-    /// * If `count` occurrences of `log_text` is not found in the log within 60 seconds.
+    /// * If `count` occurances of `log_text` is not found in the log within 60 seconds.
+    ///
+    /// # Example
+    /// ```
+    /// let _compose = DockerCompose::new("examples/redis-passthrough/docker-compose.yml")
+    ///         .wait_for("Ready to accept connections", 3);
+    /// ```
     ///
     pub fn wait_for_n(self, log_text: &str, count: usize) -> Self {
-        info!("wait_for_n: '{}' {}", log_text, count);
+        self.wait_for_n_t(log_text, count, 60)
+    }
+
+    /// Waits for a string to appear in the docker-compose log output `count` times within `time` seconds.
+    ///
+    /// Counts the number of items returned by `regex.find_iter`.
+    ///
+    /// # Arguments
+    /// * `log_text` - A regular expression defining the text to find in the docker-container log
+    /// output.
+    /// * `count` - The number of times the regular expression should be found.
+    /// * `time` - The number of seconds to wait for the count to be found.
+    ///
+    /// # Panics
+    /// * If `count` occurances of `log_text` is not found in the log within `time` seconds.
+    ///
+    /// # Example
+    /// ```
+    /// let _compose = DockerCompose::new("examples/redis-passthrough/docker-compose.yml")
+    ///         .wait_for("Ready to accept connections", 3, 65);
+    /// ```
+    pub fn wait_for_n_t(self, log_text: &str, count: usize, time: u64) -> Self {
```

Comment:
"Since we have the code" is not good reasoning.
Feel free to keep it around in a local branch for future reference if you think we might need it later.
But I do not want needless extra complexity in main

Reply:
I dont see why we couldnt just change the constant here?

Reply:
Ok, there is value in being able to set an appropriate timeout for each type of docker setup we have to deal with.

In order to actually get value out of it please:
*  change the default value from 90 to 30.
        - The only reason it was originally bumped from 30 to 90 was to support cassandra.
        - 30 is plenty for all our redis setups.
* The cassandra test is currently on 300 which is excessive, 90 should be enough.